Update order reader url

malinajirka opened the pull request:
<!-- Remember about a good descriptive title. -->


<!-- Id number of the GitHub issue this PR addresses. -->

### Description
<!-- Take the time to write a good summary. Why is it needed? What does it do? When fixing bugs try to avoid just writing “See original issue” – clarify what the problem was and how you’ve fixed it. -->
Goal of this PR is to replace "Order Card Reader" url https://woocommerce.com/products/bbpos-chipper2xbt-card-reader with https://woocommerce.com/in-person-payments/. We plan to add support for M2 readers and the generic url will allow us to update the webpage without the need to release a new version of the app.

### Testing instructions
<!-- Step by step testing instructions. When necessary break out individual scenarios that need testing, consider including a checklist for the reviewer to go through. -->
1. Tap on APp settings
2. Tap on In Person Payments
3. Tap on Order Card Reader
4. Notice the update url is loaded

### Images/gif
<!-- Include before and after images or gifs when appropriate. -->


- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

<!-- Pull request guidelines: https://github.com/woocommerce/woocommerce-android/blob/develop/docs/pull-request-guidelines.md -->

malinajirka committed:
Update order reader url

AnirudhBhat commented:
Thank you for working on this @malinajirka. I've tested this and it works fine! The code looks good as well.

❓ I see that we have tests for checking if clicking on `Order Card Reader` triggers an external web view. I'm thinking of adding a separate test for verifying the URL matches as well. Similar to what we have done for `Card Reader Manual`. WDYT?

This is not a blocker and please go ahead and merge this PR if you think adding tests can be done later in a separate PR.

malinajirka committed:
Add tests for order card reader url

malinajirka commented:
Thanks for the review @AnirudhBhat! I've added the unit test, even though I'm not sure it has any value - `NavigateToPurchaseCardReaderFlow` is an object/singleton and the url is a constant. But I guess the test could catch an issue if we changed the event into a data class.
